File: AircraftWar/src/main/scala/edu/hitsz/bullet/EnemyBullet.scala
package edu.hitsz.bullet

import edu.hitsz.animate.AnimateContainer
import edu.hitsz.application.ImageResource
import edu.hitsz.basic.PositionType.Position
import edu.hitsz.basic.Vec2Double

/**
 * @author chiro2001
 */
class EnemyBullet(posInit: Position, animateContainer: AnimateContainer[Vec2Double], power: Int)
  extends AbstractBullet(posInit, animateContainer, power)
  with ImageResource {
  override def getImageCachedPath = "images/bullet_enemy.png"
}

File: base/src/main/java/edu/hitsz/bullet/EnemyBullet.java
package edu.hitsz.bullet;

import edu.hitsz.animate.AnimateContainer;
import edu.hitsz.vector.Vec2;

/**
 * @author hitsz
 */
public class EnemyBullet extends BaseBullet {
    public EnemyBullet(Vec2 posInit, AnimateContainer animateContainer, int power) {
        super(posInit, animateContainer, power);
    }
}
